workers/data_refinery_workers/processors/test_utils.py

```diff
@@ -2,50 +2,63 @@
 import os
 from unittest.mock import MagicMock
 from django.test import TestCase
-from data_refinery_models.models import (
+from data_refinery_common.models import (
     SurveyJob,
     Batch,
     BatchStatuses,
+    File,
     ProcessorJob,
 )
 from data_refinery_workers.processors import utils
-from data_refinery_common import file_management
 
 
-def init_batch():
+def init_objects():
```

Comment:
I noticed that `init_objects()` function are available in all test modules in this PR and they look similar. How easy is it to merge them into one function?

Reply:
They are indeed similar, but the differences between them are pretty specific to the test files they're located in. Merging them into a single function might reduce the total number of lines, but I think the amount of configuration the function would have to accept to be able to meet all the different needs would end up making it much less clear. For example the `init_objects()` function in `test_utils.py` inserts two batches, whereas it only inserts one in the other two. The other two have some key differences in values for the objects it creates that are tied to the locations that the files the tests operate upon. 

Also I think that it's somewhat nice to have the initialization function defined in the test file itself so that when you work on the tests you can see what's been set up for the test without going to another file. It's not a huge benefit on it's own, but combined with the finagling that would be involved to merge the functions I think it's clearer and simpler to do keep these functions separate.